### ISSUE TITLE
Inadvertent reference to the n-word

### ISSUE THREAD

**Issue by Jookia** [vim/vim#4654]
In section 27.8 of the manual, unfortunate wording reads 'the n word' which inadvertently references a slur:

```
Vim can find a pattern that includes a line break.  You need to specify where
the line break happens, because all items mentioned so far don't match a line
break.
   To check for a line break in a specific place, use the "\n" item: >

	/the\nword

This will match at a line that ends in "the" and the next line starts with
"word".  To match "the word" as well, you need to match a space or a line
break. The item to use for it is "\_s": >
```

To reproduce this, open Vim and run ':help 27.8'. You can also read it here:
https://github.com/vim/vim/blob/d473c8c101262702ea9eeb14907ee20a786942b2/runtime/doc/usr_27.txt#L470

**Comment by k-takata** [comment]
Isn't it clear with `use the "\n" item:`?
Any proposal?

**Comment by Jookia** [comment]
Maybe something like /line\nbreak ?

**Comment by dpelle** [comment]
We could change the example as you suggest
since it does not harm to say `/line\nbreak` instead.

But personally I find this over the top: it should be
clear that \n means newline. If we search hard for
patterns, we always find odd coincidence. Here is
a silly one: if I rot13 encode runtime/doc/options.txt 
(with g? normal command) then the 'penc' Vim option
at line 5650 "shockingly" becomes 'crap'.

**Comment by Jookia** [comment]
I've submitted a PR that changes it from 'the\nword' to 'end\nstart'

**Comment by junklogar** [comment]
Are you people serious?

**Comment by chrisbra** [comment]
closed by #4669
